package.json

```diff
@@ -75,6 +75,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.7",
     "chart.js": "^2.9.4",
-    "react-chartjs-2": "^2.11.1"
+    "react-chartjs-2": "^2.11.1",
+    "reselect": "^4.0.0"
```

Comment:
The `package-lock.json` changes look fairly significant; is it possible there is an earlier version of `reselect` that is more relevant to the other packages that we use?

Reply:
Hmm. The latest release for reselect was in [version 4.0.0](https://github.com/reduxjs/reselect/releases) in Sept. 2018. It's possible there is an older version that wouldn't require bumping as many dependencies. I'm kind of tempted to bring everything else up to date. Stay tuned... going to crowdsource some opinions.

Reply:
aha! The package-lock.json file was so big because it was new! We didn't have a package-lock.json file before this change. I uninstalled and then re-installed reselect with yarn and the yarn.lock file looks completely reasonable now. 

Can you remind me if it was intentional to only manage packages with yarn rather than npm and if so, why?

Reply:
Oh, my bad for not noticing that!  IIRC, we switched to yarn in the main repo long ago because we wanted a lock file, and at that time npm didn't offer that functionality.